src/index.ts

```diff
@@ -160,11 +154,12 @@ async function startApp() {
       },
     })
 
-    server.applyMiddleware({ app, path: "/" })
+    server.applyMiddleware({ app, path })
   } else {
     const graphqlHTTP = require("express-graphql")
     app.use(
-      graphqlHTTP((req, res, params) => {
+      graphqlHTTP((req, res /*, params */) => {
+        console.log("Request from", path)
```

Comment:
Stray loggy?

Reply:
Somewhat, yeah. It does tell if the request is coming from v1 or v2 which is useful for the logs.